dolphinscheduler-common/src/main/resources/common.properties

```diff
@@ -21,7 +21,7 @@ data.basedir.path=/tmp/dolphinscheduler
 # resource view suffixs
 #resource.view.suffixs=txt,log,sh,bat,conf,cfg,py,java,sql,xml,hql,properties,json,yml,yaml,ini,js
 
-# resource storage type: LOCAL, HDFS, S3, OSS, GCS, ABS, NONE. LOCAL type is a specific type of HDFS with "resource.hdfs.fs.defaultFS = file:///" configuration
```

Comment:
Why was NONE removed? Is storage required even if I'm just using K8S tasks?

Reply:
We will use `LOCAL` type by default, you can see the above discussion. In this way, we can remove null storage plugin check for resource CURD api. In your scenario, you can ignore this configuration.

Reply:
Yes, if you don't need to use resource center, you don't need to take care of this configuration.